pom.xml

```diff
@@ -15,7 +15,7 @@
   </licenses>
 
   <artifactId>github-api</artifactId>
-  <version>1.72.2-SNAPSHOT</version>
+  <version>1.75-SNAPSHOT</version>
```

Comment:
@amuniz There is a relation between `github-api-plugin` and `github-api` (library).